code_search/src/code_search/dataflow/do_fns/prediction_do_fn.py

```diff
@@ -0,0 +1,216 @@
+"""A cherry-pick of PredictionDoFn from Kubeflow Batch Predict.
+
+DONOT submit revisions to this file. See kubeflow/batch-predict#10
```

Comment:
Can you update this with a TODO that points to an issue.

Reply:
Do you want me to create a new issue within examples? I've pointed to the batch prediction issue https://github.com/kubeflow/batch-predict/issues/10.